README.md

```diff
@@ -107,19 +107,21 @@ This repo is a mono repo that consists of 3 projects:
 
    You can install nvm from [here](https://github.com/nvm-sh/nvm).
 
-4. Set up your `.env` file
+   You also need to have pnpm installed,you can install from [here](https://pnpm.io/installation#using-npm)
+
+5. Set up your `.env` file
 
    - Go to `apps/classroomio-com` and `apps/dashboard`
    - Duplicate the `.env.example` file and rename it to `.env`
 
-5. Setup Supabase.
+6. Setup Supabase.
 
    - Make sure you've downloaded the [Supabase cli](https://github.com/supabase/cli)
    - Install and Start [docker](https://docs.docker.com/engine/install/)
    - Go to the project directory in your terminal and start Supabase
 
      ```bash
-       supabase start
+       pnpm supabase start
```

Comment:
If we change this to `pnpm`, then you also need to add the update to the package.json file with `supabase` as a dev dependency

Reply:
i dont fully understand

Reply:
oh you mean this 
```
pnpm i supabase --save-dev -w
```